commons/src/main/java/com/powsybl/commons/datastore/DataStoreUtil.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.commons.datastore;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NotDirectoryException;
+import java.nio.file.Path;
+import java.util.Objects;
+
+/**
+ * @author Giovanni Ferrari <giovanni.ferrari at techrain.eu>
+ */
+public interface DataStoreUtil {
```

Comment:
We should make it a `public final class` with private constructor instead of interface, to make clear it's a utility class.

Reply:
Ah also, i would be nice to rename it `DataStores`, it sounds more "modern" :)